connect/templates/camunda7/src/test/resources/expected-test-element-template.json

```diff
@@ -22,32 +23,32 @@
       "name" : "camunda:topic"
     }
   }, {
-    "value" : "",
-    "label" : "name",
+    "value" : "${}",
```

Comment:
müsste value hier nicht gesetzt sein?

Reply:
Das habe ich auch gedacht, aber es macht schon Sinn das leer zu lassen, da der, der den Worker schreibt nicht unbedingt weiß welche Prozessvariablen existieren.
Auch das Beispiel, dass du mir geschickt hast mit `${JSON[]}` sollte besser explizit vom Prozessentwickler gesetzt werden.

Reply:
Ja, das mit `${JSON[]}` stimmt. Das muss der user setzen.
Ich habs bisher immer so gehandhabt, dass ich meine Java Objekte nach den Prozess Variablen genannt hab. Daher würd ich das schon vorausfüllen.
Zusätzlich hats den Vorteil, dass wenn ich dadurch in einem ServiceTask ein Objekt in den Prozess geben kann, das ich dann im nächsten wieder raushol. Daher würd ich auch die Variablen nicht mit Result extenden